scripts/blueprints/os-arch.yaml

```diff
@@ -1,21 +1,22 @@
 selector:
   sw.os: 1
   arch.sw: 1
+  sw.stack-variant: 1
   sw.blob: '1+'
 output:
   filename: Dockerfile
   type: sw.docker-image.os-arch
-  slug: "{{children.arch.sw.slug}}-{{children.sw.os.slug}}:{{children.sw.os.version}}"
-  path: "{{children.sw.os.slug}}/{{children.arch.sw.slug}}/{{children.sw.os.version}}"
+  slug: "{{this.children.arch.sw.slug}}-{{this.children.sw.os.slug}}"
```

Comment:
So we're not storing the version in the slug anymore? (i.e. after the `:`)? I see that we still have it on the path. Will the deploy scripts handle it?

Reply:
@jviotti, this script will handle that task https://github.com/resin-io-library/base-images/pull/480/files#diff-ec6a48beb8b00bd417d941a15b908c8e. It will generate all possible tags for a specific image in which will be used as input for the deploy scripts.